cmd/kubeadm/app/phases/markcontrolplane/markcontrolplane.go

```diff
@@ -55,6 +56,7 @@ func taintExists(taint v1.Taint, taints []v1.Taint) bool {
 
 func markControlPlaneNode(n *v1.Node, taints []v1.Taint) {
 	n.ObjectMeta.Labels[constants.LabelNodeRoleMaster] = ""
+	n.ObjectMeta.Labels[constants.LabelNodeRoleControlPlane] = ""
```

Comment:
nit: Possibly add a comment that Role master will be removed according to the KEP